tests/metrics/zh/test_reference_based_text_quality.py

```diff
@@ -0,0 +1,120 @@
+
+
+@pytest.mark.parametrize('generated_outputs,reference_outputs',
+                         [("这样的姑娘是受不了的。", "您到底有什么事？"),
+                          (["这样的姑娘是受不了的。"], ["您到底有什么事？"])])
+@parametrize_rouge_function
+@parametrize_tokenizer
+def test_rouge_no_overlap(generated_outputs: str, reference_outputs: str,
+                          rouge_function: Callable[[str, str],
+                                                   MetricValue[float]],
+                          tokenizer: Optional[_ChineseTokenizer]) -> None:
+    # All ROUGE scores are 0 if the generated and reference outputs have no
+    # overlapping words
+    actual_metric_value = rouge_function(
+        generated_outputs,
+        reference_outputs,
+        tokenizer=tokenizer()  # type: ignore[reportGeneralTypeIssues]
+        if tokenizer else None)
+    assert actual_metric_value.metric_values == [0.]
+    assert actual_metric_value.language == 'zh'
+
+
+@pytest.mark.parametrize('generated_outputs,reference_outputs',
+                         [("床前明月光，下一句是什么？", "床前明月光的下一句是什么？"),
+                          (["床前明月光，下一句是什么？"], ["床前明月光的下一句是什么？"])])
+@parametrize_rouge_function
+@parametrize_tokenizer
+def test_rouge_some_overlap(generated_outputs: str, reference_outputs: str,
+                            rouge_function: Callable[[str, str],
+                                                     MetricValue[float]],
+                            tokenizer: Optional[_ChineseTokenizer]) -> None:
+    expected_value = {
+        'rouge1': [0.941176],
+        'rouge2': [0.8],
+        'rougeL': [0.941176]
+    }
+    # The ROUGE-2 score is lower than the ROUGE-1 and ROUGE-L scores
+    actual_metric_value = rouge_function(
+        generated_outputs,
+        reference_outputs,
+        tokenizer=tokenizer()  # type: ignore[reportGeneralTypeIssues]
+        if tokenizer else None)
+    is_close(actual_metric_value.metric_values,
+             expected_value[rouge_function.__name__])
+    assert actual_metric_value.language == 'zh'
+
+
+@pytest.mark.parametrize('generated_outputs,reference_outputs',
+                         [("那里有一本三体小说。", "那里有一本三体小说。"),
+                          (["那里有一本三体小说。"], ["那里有一本三体小说。"])])
+def test_semantic_similarity_identical(generated_outputs, reference_outputs):
+    metric_value = semantic_similarity(generated_outputs, reference_outputs)
+    assert 0.99 <= metric_value <= 1
+
+
+@pytest.mark.parametrize(
+    'generated_outputs,reference_outputs',
+    [("php是世界上最好的语言，学计算机要从娃娃抓起。", "在石家庄，有一支摇滚乐队，他们创作了很多音乐。"),
+     (["php是世界上最好的语言，学计算机要从娃娃抓起。"], ["在石家庄，有一支摇滚乐队，他们创作了很多音乐。"])])
+def test_semantic_similarity_not_similar(generated_outputs, reference_outputs):
+    # hard to find two sentence similiar < 0.25 using current model
+    # if '。' in one of the sentence pair be deleted, it would < 0.25 much easier
+    metric_value = semantic_similarity(generated_outputs, reference_outputs)
+    assert 0.0 <= metric_value <= 0.36
```

Comment:
```suggestion
    assert 0.0 <= metric_value <= 0.5
```
nit: I think it's fine to give a bit of leeway here